client/src/app/slides/motions/motion/motions-motion-slide-data.ts

```diff
@@ -1,3 +1,16 @@
 export interface MotionsMotionSlideData {
+    identifier: string;
     title: string;
+    text: string;
+    amendment_paragraphs: object[];
+    is_child: boolean;
+    show_meta_box: boolean;
+    reason: string;
+    modified_final_version: string;
+    state: string;
+    state_extension: string;
+    recommendation: string;
+    recommendation_extension: string;
+    submitter: object[];
```

Comment:
@FinnStutzenstein is this type `object[]` correct?

Reply:
Nope. Please completly specify the type
I think the type is currently
{title:string;
firstname:string;
lastname:string;}[]

Also more things: not everything is always send by the server: the reason might not be present, if the config "show reason" is disabled. Mark all fieds with a '?', that might be missing. Check the servercode. All fields that have an 'if' around it are optional. In the slide you always need to check, if some data is present before using it.